inference-engine/tests/functional/plugin/cpu/single_layer_tests/pooling.cpp

```diff
@@ -4,77 +4,115 @@
 
 #include "ngraph_functions/builders.hpp"
 #include "test_utils/cpu_test_utils.hpp"
-#include "shared_test_classes/single_layer/pooling.hpp"
 #include "test_utils/fusing_test_utils.hpp"
+#include "shared_test_classes/single_layer/pooling.hpp"
+#include "shared_test_classes/base/ov_subgraph.hpp"
 
-using namespace InferenceEngine;
 using namespace CPUTestUtils;
-using namespace LayerTestsDefinitions;
 
 namespace CPULayerTestsDefinitions {
-typedef std::tuple<
-        poolLayerTestParamsSet,
-        CPUSpecificParams,
-        fusingSpecificParams
-> poolLayerCpuTestParamsSet;
+
+using PoolingInputShapes = std::pair<std::vector<ov::PartialShape>, std::vector<ov::Shape>>;
+
+using poolLayerCpuTestParamsSet = std::tuple<LayerTestsDefinitions::poolSpecificParams,
+                                             PoolingInputShapes,
+                                             InferenceEngine::Precision,
```

Comment:
Now `ov::test::ElementType` is used, so I would say we have to change the test using this type for precision description for consistency.